integration_tests/__tests__/clear_cache.test.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+const runJest = require('../runJest');
+const skipOnWindows = require('../../scripts/skip_on_windows');
+const rimraf = require('rimraf');
+
+const CACHE = path.resolve(os.tmpdir(), 'clear_cache_directory');
+
+skipOnWindows.suite();
```

Comment:
Necessary, or copypasta? (Just asking, fine if needed :smile:)

Reply:
`skipOnWindows`? I assumed it was part of the test convention as I saw it everywhere. I can remove it if it isn't necessary.